### ISSUE TITLE
Allow customization of JDK

### ISSUE THREAD

**Issue by recampbell** [buildhive/buildhive#3]
We use JDK 1.6 by default, it should probably be 1.7?

At any rate, people should be able to specify a JDK (or an array of JDKs?)

**Comment by nelsonsilva** [comment]
"Shell Script" : JAVACMD=/opt/jdk/jdk1.7.0/bin/java ant 
Is a possible workaround for now.

**Comment by jglick** [comment]
Agreed that 7u5 should be the default for now. It is easy enough to set `-source 5` or similar if you want, and at least for Maven users Animal Sniffer makes it simple to define a baseline platform API version in your sources without actually switching JDKs.

**Comment by alexandrenavarro** [comment]
I also agree that jdk 1.7. I'm under maven and I don't understand where in jenkins I can set "Shell Script" : JAVACMD=/opt/jdk/jdk1.7.0/bin/java mvn . Is it available only custom project (non maven notably)?

**Comment by jglick** [comment]
@alexandrenavarro, right for a Maven project you cannot use that workaround.

**Comment by alexandrenavarro** [comment]
Some news on the possibility to use jdk 1.7 in maven on buildhive?

**Comment by old-o** [comment]
It is relatively easy for Maven builds:
http://dev-snapshot.blogspot.ca/2012/08/use-java-7-on-buildhive.html

**Comment by alexandrenavarro** [comment]
Thanks @odoepner I didn't realize I can do that.
